infrastructure/pyd/make_object.d

```diff
@@ -33,6 +33,8 @@ module pyd.make_object;
 
 import deimos.python.Python;
 
+import mir.ndslice.slice;
```

Comment:
Can this import be moved to just before where it is required?

Reply:
Done